ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -293,7 +293,7 @@ export default defineComponent({
             (
               !hasMinimumGroups() ||
               hasEmptyGroup.value ||
-              !hasUnsavedChanges.value ||
+              hasUnsavedChanges.value ||
```

Comment:
I think this explicitly done, for transfer flow. 
When a user tries to proceed without having any changes, it would prompt a validation error, thats where i created this bug in the Registration flow i believe. 

![Screen Shot 2023-03-06 at 2 44 06 PM](https://user-images.githubusercontent.com/53542131/223270205-4871b4e1-67c8-4ff1-868f-d167d612b015.png)


You'll need to update the condition here to handle that.

Reply:
ie `(props.isMhrTransfer && !hasUnsavedChanges.value) ||` or something along those lines.

Reply:
Ohhhh gotcha. Ill make that exclusive to registrations